ui.R

```diff
@@ -213,8 +213,16 @@ navbarPage("Power Implications for Reduced Sample Size",
            # Group Sequential Design (end)
            # ***************************************************
            tabPanel("Help",
-                    fluidPage(
-                      fluidRow(column (7, withMathJax(includeMarkdown("help.md"))))
+                    fluidPage(fluidRow(column (8, withMathJax(includeMarkdown("help_general.md"))))),
+                    tabsetPanel(

```

Comment:
I quite like the tab setting because of the long description. But the current view is not ideal since these two tabs are at the bottom of the page. So I am wondering if we could have three tabs: Overview, Power Evaluation, Group Sequential Design, so that the last two have a more prominent place.

Also your name should definitely be added! :)